ui/src/CurrentUser.ts

```diff
@@ -88,10 +88,11 @@ export class CurrentUser {
                 this.user = passThrough.data;
                 this.loggedIn = true;
                 this.hasNetwork = true;
+                this.reconnectTime = 7500;
                 return passThrough;
             })
             .catch((error: AxiosError) => {
-                if (!error || !error.response) {
+                if (!error || !error.response || error.response.status === 503) {
```

Comment:
503 doesn't mean that there is no network connection. So `this.lostNetwork()` is a little misleading here.

I'd say, we rename the variable `CurrentUserStore#hasNetwork` to `connectionError: null | string`. 
`null` means no error -> no banner and if an error is set then the banner will be shown with the message.
503 could be something like `The server couldn't successfully handle our request, (statusCode: 503).`.

Both this.lostNetwork and the NetworkLostBanner need a new name as it is not exclusive to  network lost errors anymore.